### DISCUSSION TITLE
Timeline width

### DISCUSSION THREAD

**Discussion by IsmailYaman** [saadeghi/daisyui#2966] [Q&A] [answered]
I'm using the timeline component but I can't make it the full width of the parent component. I have tried everything, but I can't seem to fix it. 

![image](https://github.com/saadeghi/daisyui/assets/25452426/ef69c59e-5e23-4cea-8c73-d7a9cce6dabc)

**Comment by saadeghi** [comment, marked as answer]
You can add `grow` class to all `<li>` tags to make them all grow and fill the space.  
Or you can add `[&>li]:grow` to `timeline` to do the same with one class.  
Example: https://play.tailwindcss.com/Rx00Jo619h

Let me know if you have a question.
